packages/sdk/src/sdk.ts

```diff
@@ -204,13 +265,37 @@ export class NxtpSdkError extends NxtpError {
       txError?: NxtpErrorJson;
       signerError?: NxtpErrorJson;
       messagingError?: NxtpErrorJson;
-      auctionError?: string;
+      auctionError?: NxtpErrorJson;
+      encryptionError?: string;
     },
   ) {
     super(message, context, TransactionManagerError.type);
   }
 }
 
+/**
+ * This is only here to make it easier for sinon mocks to happen in the tests. Otherwise, this is a very dumb thing.
+ *
+ */
+export const sdkSignFulfillTransactionPayload = async (
```

Comment:
We should probably keep it as the same name as the other function and do something like 
```ts
import { signFulfillTransactionPayload as _signFulfillTransactionPayload } from "@connext/nxtp-utils"
```

Reply:
yeah make sense

Reply:
done